packages/dev/docs/src/docs.css

```diff
@@ -889,3 +889,33 @@ h2.sectionHeader {
 .popover {
   @mixin small-propTable;
 }
+
+.index {
+  h3 {
+    margin-bottom: var(--spectrum-global-dimension-size-200) !important;
```

Comment:
I can't review this without saying, "I see this" =P

Reply:
I was able to get rid of it :)